<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Database</class>
 <widget class="QDialog" name="Database">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>651</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Database</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gui_icons.qrc">
    <normaloff>:/icons/openrpgmaker.png</normaloff>:/icons/openrpgmaker.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">.QFrame
{
	border-width: 1px;
	border-style: solid;
	border-color: palette(window-text);
}

.QFrame&gt;QLabel
{
	border-width: 0px;
	background-color: palette(window-text);
	color: palette(window);
	font-weight: bold;
}

.QFrame&gt;QCheckBox
{
	background-color: palette(window-text);
	border-width: 0px;
	border-style: none;
	color: palette(window);
	spacing: 3px;
	font-weight: bold;
	padding: 0px 0px 0px 3px;
}

.QFrame&gt;QCheckBox::indicator
{
	min-width: 8px;
	max-width: 8px;
	min-height: 8px;
	max-height: 8px;
	width: 8px;
	height: 8px;
	border-width: 2px;
	border-radius: 0px;
	border-style: ridge;
	border-color: palette(mid);
}

.QFrame&gt;QCheckBox::indicator:unchecked
{
	background-color: palette(base);
}

.QFrame&gt;QCheckBox::indicator:checked
{
	background-color: palette(window-text);
}

.QFrame&gt;QCheckBox::indicator:hover
{
	border-color: palette(highlight);
}</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Section</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="topRowButtons">
           <property name="spacing">
            <number>3</number>
           </property>
           <item>
            <widget class="QPushButton" name="bTilesets">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Tilesets</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bTerrain">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Terrain</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bCharacters">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Characters</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bClasses">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Classes</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bCharacterSprites">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Character Sprites</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bSkills">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Skills</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bItems">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Items</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bBattleAnimations">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Battle Animations</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bGlobalAnimations">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Global Animations</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="bottomRowButtons">
           <property name="spacing">
            <number>3</number>
           </property>
           <item>
            <widget class="QPushButton" name="bAttributes">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Attributes</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bConditions">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Conditions</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bStrings">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Strings</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bSystem">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>System</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bCommonEvents">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Common Events</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bMonsterAnimations">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Monster Animations</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bMonsters">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Monsters</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bMonsterGroups">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Monster Groups</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bScreenLayouts">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Screen Layouts</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>892</width>
       <height>539</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>892</width>
       <height>539</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="Database_Tilesets" name="wTilesets"/>
     <widget class="Database_Terrain" name="wTerrain"/>
     <widget class="Database_Characters" name="wCharacters"/>
     <widget class="Database_CharacterSprites" name="wCharacterSprites"/>
     <widget class="Database_Items" name="wItems"/>
     <widget class="Database_Attributes" name="wAttributes"/>
     <widget class="Database_Strings" name="wStrings"/>
     <widget class="Database_Monsters" name="wMonsters"/>
     <widget class="Database_Classes" name="wClasses"/>
     <widget class="Database_Skills" name="wSkills"/>
     <widget class="Database_Conditions" name="wConditions"/>
     <widget class="Database_System" name="wSystem"/>
     <widget class="Database_CommonEvents" name="wCommonEvents"/>
     <widget class="Database_MonsterGroups" name="wMonsterGroups"/>
     <widget class="Database_BattleAnimations" name="wBattleAnimations"/>
     <widget class="Database_GlobalAnimations" name="wGlobalAnimations"/>
     <widget class="Database_ScreenLayouts" name="wScreenLayouts"/>
     <widget class="Database_MonsterAnimations" name="wMonsterAnimations"/>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="aShiftUp">
   <property name="icon">
    <iconset resource="gui_icons.qrc">
     <normaloff>:/icons/uparrow2.png</normaloff>:/icons/uparrow2.png</iconset>
   </property>
   <property name="text">
    <string>Shift Up</string>
   </property>
  </action>
  <action name="aShiftDown">
   <property name="icon">
    <iconset resource="gui_icons.qrc">
     <normaloff>:/icons/downarrow2.png</normaloff>:/icons/downarrow2.png</iconset>
   </property>
   <property name="text">
    <string>Shift Down</string>
   </property>
  </action>
  <action name="aShiftLeft">
   <property name="icon">
    <iconset resource="gui_icons.qrc">
     <normaloff>:/icons/leftarrow2.png</normaloff>:/icons/leftarrow2.png</iconset>
   </property>
   <property name="text">
    <string>Shift Left</string>
   </property>
  </action>
  <action name="aShiftRight">
   <property name="icon">
    <iconset resource="gui_icons.qrc">
     <normaloff>:/icons/rightarrow2.png</normaloff>:/icons/rightarrow2.png</iconset>
   </property>
   <property name="text">
    <string>Shift Right</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>Database_Tilesets</class>
   <extends>QWidget</extends>
   <header>database_tilesets.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_Terrain</class>
   <extends>QWidget</extends>
   <header>database_terrain.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_Characters</class>
   <extends>QWidget</extends>
   <header>database_characters.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_Items</class>
   <extends>QWidget</extends>
   <header>database_items.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_Attributes</class>
   <extends>QWidget</extends>
   <header>database_attributes.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_Strings</class>
   <extends>QWidget</extends>
   <header>database_strings.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_Monsters</class>
   <extends>QWidget</extends>
   <header>database_monsters.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_Classes</class>
   <extends>QWidget</extends>
   <header>database_classes.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_Skills</class>
   <extends>QWidget</extends>
   <header>database_skills.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_Conditions</class>
   <extends>QWidget</extends>
   <header>database_conditions.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_System</class>
   <extends>QWidget</extends>
   <header>database_system.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_CommonEvents</class>
   <extends>QWidget</extends>
   <header>database_commonevents.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_MonsterGroups</class>
   <extends>QWidget</extends>
   <header>database_monstergroups.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_BattleAnimations</class>
   <extends>QWidget</extends>
   <header>database_battleanimations.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_GlobalAnimations</class>
   <extends>QWidget</extends>
   <header>database_globalanimations.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_MonsterAnimations</class>
   <extends>QWidget</extends>
   <header>database_monsteranimations.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_ScreenLayouts</class>
   <extends>QWidget</extends>
   <header>database_screenlayouts.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Database_CharacterSprites</class>
   <extends>QWidget</extends>
   <header>database_charactersprites.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="gui_icons.qrc"/>
 </resources>
 <connections/>
</ui>
